src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs

```diff
@@ -75,7 +75,7 @@ public string DisplayName
         public DateTime LastUpdated => Revisions.LastOrDefault()?.CreationDate ?? CreationDate;
 
         [JsonIgnore]
-        public string Language => Revisions.FirstOrDefault()?.Files.FirstOrDefault()?.Language;
+        public string Language => Revisions.LastOrDefault()?.Files.LastOrDefault()?.Language;
```

Comment:
Is using the last file important?

Reply:
There were few python reviews which initially updated as json and then created new revision with wheel. In general I think its better to refer latest value even though most of the reviews will have same value for first and last revision.